src/ControlPanel.jsx

```diff
@@ -106,6 +106,7 @@ class ControlPanel extends Component {
 ControlPanel.propTypes = {
   relay: propTypes.element.isRequired,
   filter: propTypes.element.isRequired,
+  stop: propTypes.element.isRequired,
```

Comment:
Could we make these names more intuitive? Could we leave them to `toggleStops`, `filterStops`?

Reply:
Good point, i'll change it